### ISSUE TITLE
[Refactor] Refactor > Extract > Flutter widget should be easier to access.  

### ISSUE THREAD

**Issue by johnsonmh** [flutter/flutter-intellij#3985]
## Steps to Reproduce

This feature is very very useful, but hard to access. 
There's apparently a shortcut, but it does not work for me on Android Studio on MacOS.

Right now, it's here:

![giphy-1](https://user-images.githubusercontent.com/4229329/67031126-e2ccf680-f0de-11e9-9191-a55007b7f2b1.gif)

I would want it to be one of these options:

![Screen Shot 2019-10-17 at 1 06 19 PM](https://user-images.githubusercontent.com/4229329/67031156-f24c3f80-f0de-11e9-9653-b61f42d92751.png)

**Comment by jacob314** [comment]
I agree it should be in this list. We should also consider creating custom Flutter UI displaying the list of refactors to perform when a Widget or Widget property is selected.
